### ISSUE TITLE
Wordpress not starting

### ISSUE THREAD

**Issue by sphimmer** [bitnami/bitnami-docker-wordpress-nginx#9]
**Description**

I was getting this issue when following your readme to get the docker-compose.yml running.
`wordpress_1  | 2019/10/01 13:04:14 [crit] 109#0: pread() "/opt/bitnami/nginx/conf/server_blocks/wordpress-server-block.conf" failed (21: Is a directory)
wordpress_1  | nginx: [crit] pread() "/opt/bitnami/nginx/conf/server_blocks/wordpress-server-block.conf" failed (21: Is a directory)
`
it was creating `wordpress-server-block.conf` ` as a directory. So i ran `touch wordpress-server-block.conf` after deleting it. it now boots up but gives me the nginx welcome screen. how do i get wordpress to start up? seems like something is missing from the documentation.

**Steps to reproduce the issue:**

1. $ curl -LO https://raw.githubusercontent.com/bitnami/bitnami-docker-wordpress-nginx/master/5/debian-9/docker-compose.yml
2. $ docker-compose up
2. $ docker-compose down 
3. $ rm -r wordpress-server-block.conf
4. $ touch wordpress-server-block.conf
5. $ docker-compose up
6. navigate to server in browser. will only see nginx welcome screen. no wordpress.

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.2
 API version:       1.40
 Go version:        go1.12.8
 Git commit:        6a30dfc
 Built:             Thu Aug 29 05:29:11 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.2
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.8
  Git commit:       6a30dfc
  Built:            Thu Aug 29 05:27:45 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 2
  Running: 0
  Paused: 0
  Stopped: 2
 Images: 4
 Server Version: 19.03.2
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc version: 425e105d5a03fabd737a126ad93d62a9eeede87f
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-1044-aws
 Operating System: Ubuntu 18.04.2 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 983.9MiB
 Name: ip-172-31-22-88
 ID: DSCU:QKOA:OEOI:Z7NP:7RLE:5MG5:AFSG:IXT7:G6LV:CZKB:AZOZ:77MQ
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.24.1, build 4667896b
docker-py version: 3.7.3
CPython version: 3.6.8
OpenSSL version: OpenSSL 1.1.0j  20 Nov 2018
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
running on ubuntu ec2 18.04

**Comment by alemorcuq** [comment]
Hi, @sphimmer,

Does the `./wordpress-server-block.conf` file exist in your host? If not, you will have to download it too:

```console
$ curl -sLO https://raw.githubusercontent.com/bitnami/bitnami-docker-wordpress-nginx/master/5/debian-9/docker-compose.yml
$ curl -sLO https://raw.githubusercontent.com/bitnami/bitnami-docker-wordpress-nginx/master/5/debian-9/wordpress-server-block.conf
$ docker-compose up
```

Regards,
Alejandro

**Comment by Remzi1993** [comment]
Hi @sphimmer 
I have proposed a PR which should fix this issue, see: #10 
I know this is almost a month ago, but I stumbled on the same issue so I would like to fix this.

Kind regards,
Remzi
